sphinx/source/docs/install.md

```diff
@@ -175,15 +175,13 @@ Cross-compilation of ROS 2 workspaces allows to build both CPU binaries as well
 
 ###################################################
 # 1. install some dependencies you might be missing
-#
-# NOTE: gcc-multilib conflicts with Yocto/PetaLinux 2022.1 dependencies
-# so you can't have both paths simultaneously enabled in a single
-# development machine
 ###################################################
 sudo apt-get -y install curl build-essential libssl-dev git wget \
                           ocl-icd-* opencl-headers python3-vcstool \
                           python3-colcon-common-extensions python3-colcon-mixin \
-                          kpartx u-boot-tools pv gcc-multilib gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+                          kpartx u-boot-tools pv gcc-aarch64-linux-gnu g++-aarch64-linux-gnu \
+                          ros-humble-camera-info-manager ros-humble-camera-info-manager-dbgsym \ 
+                          qemu-user-static
```

Comment:
My testing here didn't require 'ros-humble-camera-info-manager' nor 'ros-humble-camera-info-manager-dbgsym' since there were brought in with the 'ros-humble-desktop-full' installation.
Although I think the installation instruction should have a few more words about 'ros-humble-desktop-**full**' being required, and not just 'ros-humble-deskop'

I do still have build issues due to ament-cmak not found in the final cross-compile stage however, so unsure about the qemu-user-static requirement.